command/executor_plugin.go

```diff
@@ -11,9 +11,17 @@ import (
 	plugin "github.com/hashicorp/go-plugin"
 
 	"github.com/hashicorp/nomad/drivers/shared/executor"
+	"github.com/hashicorp/nomad/lib/circbufwriter"
 	"github.com/hashicorp/nomad/plugins/base"
 )
 
+const (
+	// circleBufferSize is the size of the in memory ring buffer used for
+	// go-plugin logging to stderr. When the buffer exceeds this size before
+	// flushing it will begin overwriting data
+	circleBufferSize = 64 * 1024
```

Comment:
Is this size related to https://github.com/hashicorp/nomad/pull/4765 for example? we log up to 200kb log lines in some cases

Reply:
okay, thanks for clarifying @nickethier